<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCustomPlot Scrollbar Axis Range Control Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>SAMPLE READ</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>40</y>
           <width>50</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>SR</string>
         </property>
        </widget>
        <widget class="QScrollBar" name="horizontalScrollBar_3">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>110</y>
           <width>160</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QScrollBar" name="horizontalScrollBar_5">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>210</y>
           <width>160</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_3">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>200</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>90</y>
           <width>50</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>Dofs</string>
         </property>
        </widget>
        <widget class="QScrollBar" name="horizontalScrollBar_2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>60</y>
           <width>160</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_7">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>350</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_5">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>150</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QScrollBar" name="horizontalScrollBar_4">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>160</y>
           <width>160</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QScrollBar" name="horizontalScrollBar_6">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>300</y>
           <width>160</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_4">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>110</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QScrollBar" name="horizontalScrollBar_7">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>350</y>
           <width>160</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>190</y>
           <width>50</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>Vofs</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>60</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_6">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>300</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>140</y>
           <width>50</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>a</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="geometry">
          <rect>
           <x>183</x>
           <y>430</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>430</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Sampling Count</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>280</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Pulse Number</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>RESET PIXEL</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>PULSE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSAVE"/>
    <addaction name="actionEXIT"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionInsert_Plot">
   <property name="text">
    <string>Insert Plot</string>
   </property>
  </action>
  <action name="actionSave_Document">
   <property name="text">
    <string>Save Document...</string>
   </property>
  </action>
  <action name="actionSAVE">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionEXIT">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
